pkg/agent/openflow/network_policy.go

```diff
@@ -1405,7 +1405,7 @@ func parseDropFlow(flow string) (uint32, types.RuleMetric) {
 	m.Sessions = pkts
 	bytes, _ := strconv.ParseUint(segs[2][strings.Index(segs[2], "=")+1:], 10, 64)
 	m.Bytes = bytes
-	id, _ := strconv.ParseUint(segs[6][strings.Index(segs[6], "0x")+2:strings.Index(segs[6], " ")], 16, 64)
+	id, _ := strconv.ParseUint(segs[5][strings.Index(segs[5], "0x")+2:strings.Index(segs[5], " ")], 16, 64)
```

Comment:
does the example flow above (`table=101, n_packets=9, n_bytes=666, priority=200,ip,reg0=0x100000/0x100000,reg3=0x5 actions=drop`) need to be updated as well to remove the `ip` match?